### ISSUE TITLE
直接在vs下编译就行吗

### ISSUE THREAD

**Issue by wedwin** [chaowang15/rgbdrecon#1]
直接在vs下编译就行吗

**Comment by chaowang15** [comment]
@wedwin Yes it is compiled and tested in Visual Studio 2013. Actually this code is still under reconstruction. but you can still give it a try.
